src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/Players/PlayerViewModel.cs

```diff
@@ -115,15 +116,7 @@ public static PlayerViewModel Map(Player player)
         /// <returns> Domain object </returns>
         public Player ToDomain()
         {
-            return new Player {
-                Id = Id,
-                FirstName = FirstName,
-                LastName = LastName,
-                BirthYear = BirthYear,
-                Height = Height,
-                Weight = Weight,
-                TeamId = TeamId
-            };
+            return new Player(Id, FirstName, LastName, BirthYear, Height, Weight, TeamId);
```

Comment:
Will this line work when we create new player?

Reply:
No.  

![image](https://user-images.githubusercontent.com/11724508/38165001-cf2d9be2-3515-11e8-9184-7597cb668ca2.png)

![image](https://user-images.githubusercontent.com/11724508/38165011-e62ca392-3515-11e8-9039-da2ca20206f1.png)

Reply:
Do not use use ToDomain. Also consider my comment on service method